bigquery/api/Snippets/BrowseTable.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright(c) 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// [START bigquery_browse_table]
+
+using Google.Api.Gax;
+using Google.Apis.Bigquery.v2.Data;
+using Google.Cloud.BigQuery.V2;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class BigQueryBrowseTable
+{
+    public void BrowseTable(
+        string projectId = "your-project-id"
+    )
+    {
+        BigQueryClient client = BigQueryClient.Create(projectId);
+        TableReference tableReference = new TableReference()
+        {
+            TableId = "shakespeare",
+            DatasetId = "samples",
+            ProjectId = "bigquery-public-data"
+        };
+        // Load all rows from a table
+        PagedEnumerable<TableDataList, BigQueryRow> result = client.ListRows(
+            tableReference: tableReference,
+            schema: null
+        );
+        // Print the first 10 rows
+        foreach (BigQueryRow row in result.Take(10))
```

Comment:
Nice. I really like the LINQ methods in C#.

We could do the same in Python with [islice](https://docs.python.org/3/library/itertools.html#itertools.islice). I wonder if we should use that in Python examples instead of `max_results` since it's more standard Python? I guess `max_results` is here to stay in Python, even if it is confusing with an API parameter of the same name.

Reply:
I think it makes sense to use the parameter in python in our sample as long as it exists. But yes, it did send me on a wild goose chase when trying to find the equivalent parameter when looking at the different libraries 😬